bots/create-and-run-a-bot/rasa-bot/README.md

```diff
@@ -11,95 +11,93 @@ You will need Python knowledge only when creating custom actions in Rasa which c
+This guide uses a remote Rocket.Chat instance, so it is necessary to get a public URL for the Rasa bot
+to be able to link it properly. It is recommended to use [ngrok](https://ngrok.com/download) for this purpose.
 
-    [Download ngrok](https://ngrok.com/download), open terminal in the folder you downloaded ngrok to and execute the following command:
+[Download ngrok](https://ngrok.com/download), open terminal in the folder you downloaded ngrok to and execute the following command:
 
-    ```
-    ./ngrok http 5005
-    ```
+```
+./ngrok http 5005
+```
 
-    You should see the following output:
+You should see the following output:
 
-    ```
-    Session Status                online
-    Session Expires               7 hours, 59 minutes
-    Version                       2.3.35
-    Region                        United States (us)
-    Web Interface                 http://127.0.0.1:4040
-    Forwarding                    http://dde093e1.ngrok.io -> http://localhost:5005
-    Forwarding                    https://dde093e1.ngrok.io -> http://localhost:5005
-    ```
+```
+Session Status                online
+Session Expires               7 hours, 59 minutes
+Version                       2.3.35
+Region                        United States (us)
+Web Interface                 http://127.0.0.1:4040
+Forwarding                    http://dde093e1.ngrok.io -> http://localhost:5005
+Forwarding                    https://dde093e1.ngrok.io -> http://localhost:5005
+```
 
-    Public URL will be provided next to `Forwarding` column. In this case, `http://dde093e1.ngrok.io`.
+Public URL will be provided next to `Forwarding` column. In this case, `http://dde093e1.ngrok.io`.
 
-4. **configure Rocket.Chat webhook:**
+**4. configure Rocket.Chat webhook**
 
-    Go to `Administration > Integrations`. Click `New Integration` button in the top right corner and then select `Outgoing WebHook`.
+Go to `Administration > Integrations`. Click `New Integration` button in the top right corner and then select `Outgoing WebHook`.
 
-    Select `Message Sent` Event Trigger from the dropdown list. Complete the configuration with the following settings:
+Select `Message Sent` Event Trigger from the dropdown list. Complete the configuration with the following settings:
 
-    * Enabled: `True`
-    * Channel: `#general`
-    * URLs: `http://<ngrok_public_url>/webhooks/rocketchat/webhook`
-    * Post as: `<RASA USER NAME>`
+* Enabled: `True`
+* Channel: `#general`
+* URLs: `http://<ngrok_public_url>/webhooks/rocketchat/webhook`
+* Post as: `<RASA USER NAME>`
 
-    **NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
+**NOTE**: Make sure to replace the URL with a valid public URL obtained on the previous step using ngrok.
 
-    Click `Save changes` in the top right corner.
+Click `Save changes` in the top right corner.
 
-5. **talk to your bot**
+**5. talk to your bot**
```

Comment:
talkin_ing_ ?

Reply:
no. all step are in infinitive: clone, configure, run, talk